lua/codeium/api.lua

```diff
@@ -177,11 +177,9 @@ function Server:new()
 
 		local manager_dir = config.manager_path
 		if not manager_dir then
-			if io.get_system_info().os == "windows" then
-				manager_dir = vim.fn.stdpath("cache") .. "/codeium/manager"
+			manager_dir = io.tempdir("codeium/manager")
+			if vim.fn.has("win32") == 1 then
```

Comment:
It's safe to mkdir with `p` if the directory exists, so you can remove this `if` check.

Reply:
Ah ok. I didn't know that thanks!